sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/TelemetryPartA.cs

```diff
@@ -55,6 +60,25 @@ internal static TelemetryItem GetTelemetryItem(Activity activity, ref TagEnumera
             return telemetryItem;
         }
 
+        private static string GetOperationName(Activity activity, ref AzMonList partBTags)
+        {
+            var httpMethod = AzMonList.GetTagValue(ref partBTags, SemanticConventions.AttributeHttpMethod)?.ToString();
+            if (!string.IsNullOrWhiteSpace(httpMethod))
+            {
+                if (activity.DisplayName.StartsWith("/", System.StringComparison.OrdinalIgnoreCase))
+                {
+                    return $"{httpMethod} {activity.DisplayName}";
+                }
+                var httpTarget = AzMonList.GetTagValue(ref partBTags, SemanticConventions.AttributeHttpTarget)?.ToString();
+                if (!string.IsNullOrWhiteSpace(httpTarget))
```

Comment:
@cijothomas @reyang - Is it ok to fall back to `http.target` here?
in case of routing(asp.netcore) I observed that the displayname gets overwritten with path template which does not start with `/`. In instrumentations we will be setting `http.target` to request path [PR](https://github.com/open-telemetry/opentelemetry-dotnet/pull/2266)

Reply:
Seems like a schema mapping topic, can we follow up in the schema doc? (I guess Python might have similar problem with Django)

Reply:
Yes - I am planning to do that.

Reply:
This is ready for review now - as per https://github.com/microsoft/common-schema/pull/272